functions/text-transform/1.0/function.json

```diff
@@ -0,0 +1,65 @@
+{
+  "description": "Transforms all the letters of a string to lowercase or uppercase, changes the first letter of a string to a capital, or transforms a string to a parameter making it a URL-friendly value.",
+  "label": "Transform Text",
+  "category": "Misc",
+  "icon": "LightningIcon",
+  "options": [
+    {
+      "meta": {
+        "type": "Text",
+        "validations": {
+          "required": true
+        }
+      },
+      "name": "value",
+      "label": "Text value",
+      "info": "The value that's going to be transformed"
+    },
+    {
+      "name": "downcase",
+      "label": "Downcase",
+      "info": "Transforms all characters of a string to lowercase.",
+      "meta": {
+        "type": "Boolean"
+      }
+    },
+    {
+      "name": "capitalize",
+      "label": "Capitalize",
+      "info": "Changes the first letter of a string to a capital.",
+      "meta": {
+        "type": "Boolean"
+      }
+    },
+    {
+      "name": "upcase",
+      "label": "Upcase",
+      "info": "Transforms all the letters of a string to uppercase.",
+      "meta": {
+        "type": "Boolean"
+      }
+    },
+    {
+      "name": "parameterize",
+      "label": "Parameterize",
+      "info": "Transforms a string to a parameter, making it a URL-friendly value.",
+      "meta": {
+        "type": "Boolean"
+      }
+    },
```

Comment:
I think it's a bit confusing to be able to select multiple transformations at once. What would happend for example when I select `downcase` and `capitalize`? Will I get a capitalized string? Or will it be downcased?
Perhaps it's better to have one option `transformation` where you can select the transformation you want to use?

Reply:
I agree with the suggestion for a single option. Will look at this next week.